bootstrap/kubeadm/hack/verify-shellcheck.sh

```diff
@@ -1,54 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I don't think we are doing any shellcheck validation in the top level verify targets/scripts, do we want to lose that?

I'm not sure if there are other verification scripts we are losing here additionally.

Reply:
We can port later, can you open an issue?

Reply:
Could you create the issue(s)? I'm not sure the full scope of removals from just a PR review and would rather not have to try diff CABPK/CAPI to determine the delta.

Reply:
Sure